features/admin/employer_send_secure_message.feature

```diff
@@ -4,6 +4,7 @@ Feature: As a Super Admin I will have the ability to see and access "Send Secure
     Given the shop market configuration is enabled
     Given all announcements are enabled for user to select
     Given a ACA site exists with a benefit market
+    Given send secure message to employer is enabled
```

Comment:
Pull this into `disable_shop_base` too (make sure you bring the step from wherever the `.rb` steps file is for this too @eidarus